label_studio/core/settings/base.py

```diff
@@ -288,6 +288,14 @@
     },
 }
 
+CACHES = {
+    'default': {
+        # 'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
```

Comment:
I would use LocMemCache, because a lot of users run our app in k8s and k8s has some problems with its storage persistence.

Reply:
I was trying to avoid any memory related approaches